metaploy/postinstall.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+cleanup() {
+	echo "Container stopped. Removing nginx configuration."
+	rm /etc/nginx/sites-enabled/heimdall.metaploy.conf
+}
+
+trap 'cleanup' SIGQUIT SIGTERM SIGHUP
+
+"${@}" &
+
+cp /heimdall.metaploy.conf /etc/nginx/sites-enabled
+
+wait $!
+
+echo "lmao"
```

Comment:
much echo

Reply:
lmaoo forgot to remove it. was added for debugging.

Reply:
i thought it was a joke. doesnt matter anyway

Reply:
Today's mistake, tomorrow's easter egg.